### PR TITLE
Add Cerebro package

### PR DESCRIPTION
**Pull request by deric**
[Cerebro](https://github.com/lmenezes/cerebro) is a web interface for managing Elastic clusters.

### CODE REVIEW

File: repo/packages/C/cerebro/0/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "description": "Cerebro",
+  "framework": false,
+  "maintainer": "https://dcos.io/community",
+  "minDcosReleaseVersion": "1.9",
+  "name": "cerebro",
+  "packagingVersion": "4.0",
+  "postInstallNotes": "",
+  "postUninstallNotes": "DC/OS Cerebro service is being uninstalled.",
+  "preInstallNotes": "Cerebro is a Elasticsearch web administration tool formerly known as Kopf. Cerebro is a standalone application (not an Elasticsearch plugin) complatible with Elasticsearch 5.x and newer.",
+  "selected": true,
```
Comment (larioj) [inline repo/packages/C/cerebro/0/package.json]: Only white listed packages are allowed to set this to true :)

Comment (deric) [reply repo/packages/C/cerebro/0/package.json]: Sorry about that, I've copied that from another framework :)

**Comment by deric** [comment]
@larioj Hopefully all major issues has been fixed. The application should be able to run under unprivileged user (`nobody` by default). I was having problem with connection timeout to local SQLite database, it turned out that increasing CPU resource should fix the problem.

File: repo/packages/C/cerebro/0/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "description": "Cerebro",
+  "framework": false,
+  "maintainer": "https://dcos.io/community",
+  "minDcosReleaseVersion": "1.9",
+  "name": "cerebro",
+  "packagingVersion": "4.0",
+  "postInstallNotes": "",
+  "postUninstallNotes": "DC/OS Cerebro service is being uninstalled.",
+  "preInstallNotes": "Cerebro is a Elasticsearch web administration tool formerly known as Kopf. Cerebro is a standalone application (not an Elasticsearch plugin) complatible with Elasticsearch 5.x and newer.",
```
Comment (ryadav88) [inline repo/packages/C/cerebro/0/package.json]: Please add this line at the beginning of preInstallNotes: `This DC/OS Service is currently in preview. There may be bugs, incomplete features, incorrect documentation, or other discrepancies. Preview packages should never be used in production!`

**Comment by deric** [comment]
@ryadav88 I've made requested changes and submitted PR to examples repo. Let me know if you need anything else.

**Pull request merged**